frontend/src/metabase/components/HistoryModal.jsx

```diff
@@ -34,6 +34,15 @@ export default class HistoryModal extends Component {
     }
   }
 
+  shouldRenderRevisionEntry({ diff }) {
+    if (diff === null) {
+      return true;
```

Comment:
i'd throw a comment on here. I remember the context of why this is true but in like 3 more hours I'll completely forget.

Reply:
@dpsutton thrown.

I don't know if the are other typical situations, are there?

Reply:
editing comment above, not sure if you got a notification. copying:
> there's also the situation that once you hit >15 revisions we start dropping older ones. So there it wouldn't be an initial revision but just the last revision we keep, if you want to hint at that in the comment

Reply:
@dpsutton thanks, updated. Is that in line with what you meant?

Reply:
yeah that's perfect. that's an edge case that might be easy to forget about and isn't in testing so the dropping might be forgotten about